app/src/main/index.js

```diff
@@ -583,14 +615,15 @@ async function pickAndConnect(addressbook) {
       err
     )
     addressbook.flagNodeIncompatible(nodeIP)
+    await stopLCD()
     return await pickAndConnect(addressbook)
```

Comment:
i know this is unrelated to your PR @NodeGuy but why are we still "picking and connecting" CC @faboweb ?

Reply:
can be renamed/refactored imo. feel free to open an issue.

Reply:
ahh! your PR isn't in yet!